src/main/java/com/sarahhand/fractals/model/colorscheme/mandelbrotset/EscapeTimeColorScheme.java

```diff
@@ -38,11 +38,7 @@ public Color getColor(PointData data, FractalConfig config){
 		MandelbrotPointData castData = (MandelbrotPointData)data;
 		MandelbrotConfig castConfig = (MandelbrotConfig)config;
 		
-		if(castData.getEscapeTime() == -1){
-			return Color.black;
-		}
-		
-		double log_zn = Math.log(castData.getEndZValue().x*castData.getEndZValue().x+castData.getEndZValue().y*castData.getEndZValue().y)/2;
+		double log_zn = Math.log(castData.getZValues().get(castData.getZValues().size()-1).x*castData.getZValues().get(castData.getZValues().size()-1).x+castData.getZValues().get(castData.getZValues().size()-1).y*castData.getZValues().get(castData.getZValues().size()-1).y)/2;
```

Comment:
To simplify this expression, can you create local variable lastZValue and assign it to castData.getZValues().size()-1, and then use lastZValue in this expression?